test/unbox.test.js

```diff
@@ -71,7 +73,8 @@ test('unbox', async t => {
   t.end()
 })
 
-test('unbox - test vectors', async t => {
+// TODO: this tests envelope() which we're probably gonna remove in favor of box2, so we should remove this test too right?
```

Comment:
see discussion in envelope file

Reply:
The JS implementation needs to pull `envelope-spec/vectors` and test them _somewhere_ in JS implementations.

e.g. note https://github.com/ssbc/envelope-js/blob/master/test/unbox.test.js has tests for some these same unbox vectors. THAT could be the better home.

Make a choice which means all the test vectors are getting tested somewhere :D

Reply:
the tests themselves look pretty similar. the differences look like 

1. in tribes there's 2 unbox vector files, in envelope-spec there's 1
2. the one in the spec is just one case with `"scheme": "envelope-id-based-dm-converted-ed25519"` but the ones in tribes are all `"scheme": "envelope-large-symmetric-group"`

i think these could/should be merged into the test in envelope-js

i'll take a look at ssb-box2 if group decryption is maybe tested there

Reply:
aah hmm i think i remember now. i think the vectors here in tribes were just copy pasted from private-group-spec here https://github.com/ssbc/private-group-spec/tree/master/vectors but without the conversion to URIs that happened there at some point.

private-group-spec doesn't seem to be doing much with those vectors either.

i propose:
* [ ] remove the vectors from private-group-spec
* [ ] remove the vector test from unbox.test.js in ssb-tribes (already done)
* [ ] copy the unbox vectors from ssb-tribes to envelope-js and delete them here
* [ ] test the vectors using the already existing test in envelope-js

i'll make an issue for this since it's gonna be hard to track here

Reply:
made issue https://github.com/ssbc/ssb-tribes/issues/102